tests/10apidoc/01register.pl

```diff
@@ -61,6 +61,32 @@
       });
    };
 
+test "POST /createUser can get or create a user",
+   requires => [ $main::API_CLIENTS[0] ],
+
+   critical => 1,
```

Comment:
I have literally no idea what this flag means, but this test probably isn't. @leonerd ?